packable/packable/src/packable/prefix/btreeset.rs

```diff
@@ -0,0 +1,139 @@
+// Copyright 2021-2022 IOTA Stiftung
```

Comment:
```suggestion
// Copyright 2023 IOTA Stiftung
```

Reply:
I did them all

Reply:
Oh alright, I don't know if we should tbh. I've always been under the impression that we should update the date when we touch a file. But I guess it's fine.